File: Vigilantes.DaprWorkshop.LoyaltyService/csharp/Models/LoyaltySummary.cs
using Newtonsoft.Json;

namespace Vigilantes.DaprWorkshop.LoyaltyService.Models
{
    public class LoyaltySummary
    {
        [JsonProperty("loyaltyId")]
        public string LoyaltyId { get; set; }
        [JsonProperty("customerName")]
        public string CustomerName { get; set; }
        [JsonProperty("pointsEarned")]
        public int PointsEarned { get; set; }
        [JsonProperty("pointTotal")]
        public int PointTotal { get; set; }
    }
}

File: Vigilantes.DaprWorkshop.LoyaltyService/java/loyalty-service/src/main/java/vigilantes/daprworkshop/loyaltyservice/model/LoyaltySummary.java
package vigilantes.daprworkshop.loyaltyservice.model;

import lombok.Data;

@Data
public class LoyaltySummary {
    
    public String loyaltyId;
    
    public String customerName;
    
    public int pointsEarned;
    
    public int pointTotal;
}